templates/glance/config/00-config.conf

```diff
@@ -12,12 +12,18 @@ bind_host=0.0.0.0
 bind_port=9292
 workers=3
 image_cache_dir=/var/lib/glance/image-cache
-log_config_append=/etc/glance/logging.conf
```

Comment:
@konan-abhi removing this line as we're going to go with https://github.com/openstack-k8s-operators/glance-operator/pull/263

Reply:
Sounds good